SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -258,7 +283,23 @@ void USpatialSender::SendRPC(UObject* TargetObject, UFunction* Function, void* P
 		if (!UnresolvedObject)
 		{
 			check(EntityId > 0);
-			Connection->SendCommandRequest(EntityId, &CommandRequest, RPCInfo->Index + 1); 
+			Worker_RequestId RequestId = Connection->SendCommandRequest(EntityId, &CommandRequest, RPCInfo->Index + 1);
+
+			if (Function->HasAnyFunctionFlags(FUNC_NetReliable))
+			{
+				void* NewParameters = Parameters;
```

Comment:
Can this not be a `TUniquePtr` or `TSharedPtr`? I don't think a raw pointer with raw `new`s and `delete`s are needed.